go.mod

```diff
@@ -1,75 +1,74 @@
 module github.com/ONSdigital/dp-topic-api
 
-go 1.19
+go 1.20
 
 replace github.com/spf13/cobra => github.com/spf13/cobra v1.4.0
 
+// to fix: [CVE-2023-32731] CWE-Other
+replace google.golang.org/grpc => google.golang.org/grpc v1.55.0
+
 require (
-	github.com/ONSdigital/dp-api-clients-go v1.43.0
-	github.com/ONSdigital/dp-api-clients-go/v2 v2.187.0
-	github.com/ONSdigital/dp-authorisation v0.2.0
-	github.com/ONSdigital/dp-component-test v0.7.0
-	github.com/ONSdigital/dp-healthcheck v1.5.0
-	github.com/ONSdigital/dp-mongodb/v3 v3.3.0
-	github.com/ONSdigital/dp-net v1.5.0
-	github.com/ONSdigital/dp-net/v2 v2.6.0
-	github.com/ONSdigital/log.go/v2 v2.3.0
-	github.com/cucumber/godog v0.12.5
-	github.com/gofrs/uuid v4.2.0+incompatible
+	github.com/ONSdigital/dp-api-clients-go/v2 v2.254.0
+	github.com/ONSdigital/dp-authorisation v0.2.1
+	github.com/ONSdigital/dp-component-test v0.9.2
+	github.com/ONSdigital/dp-healthcheck v1.6.1
+	github.com/ONSdigital/dp-mongodb/v3 v3.0.0
+	github.com/ONSdigital/dp-net/v2 v2.10.0
+	github.com/ONSdigital/log.go/v2 v2.4.1
+	github.com/cucumber/godog v0.12.6
+	github.com/gofrs/uuid v4.4.0+incompatible
 	github.com/gorilla/mux v1.8.0
 	github.com/justinas/alice v1.2.0
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/pkg/errors v0.9.1
-	github.com/smartystreets/goconvey v1.7.2
-	go.mongodb.org/mongo-driver v1.9.1
+	github.com/smartystreets/goconvey v1.8.0
+	github.com/stretchr/testify v1.8.4
+	go.mongodb.org/mongo-driver v1.11.7
 )
 
 require github.com/hashicorp/go-uuid v1.0.3 // indirect
```

Comment:
why's this one by itself? did you do a `go mod tidy`?

Reply:
For some reason go mod tidy does not remove it. Had to remove it by hand.